docs/reference/ccr/apis/auto-follow/delete-auto-follow-pattern.asciidoc

```diff
@@ -5,7 +5,7 @@
 <titleabbrev>Delete auto-follow pattern</titleabbrev>
 ++++
 
-Delete auto-follow patterns.
+Delete <<xpack-ccr,{ccr}>> auto-follow patterns.
```

Comment:
this should link to the auto-follow pattern docs instead. if someone needs a ccr link they can get it from the ccr api overview page.

commenting this for visibility for future work - I think we should link to the closest conceptual doc match for the actual item the API impacts/mutates.

```suggestion
Delete {ccr} <<ccr-auto-follow,auto-follow patterns>>.
```